index.js

```diff
@@ -41,7 +47,22 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
+  let high = 0;
+  let meta = movies[0];
```

Comment:
Not sure what a `meta` is supposed to be other than a company that might currently have an very uncertain future right now, but looking at this code I'm not quite sure we need it.